server/instrumentation.js

```diff
@@ -0,0 +1,63 @@
+const { NodeSDK } = require('@opentelemetry/sdk-node')
+const {
+  getNodeAutoInstrumentations
+} = require('@opentelemetry/auto-instrumentations-node')
+const { PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics')
+const {
+  OTLPTraceExporter
+} = require('@opentelemetry/exporter-trace-otlp-proto')
+const {
+  OTLPMetricExporter
+} = require('@opentelemetry/exporter-metrics-otlp-proto')
+const {
+  ATTR_SERVICE_NAME,
+  ATTR_SERVICE_VERSION
+} = require('@opentelemetry/semantic-conventions')
+const {
+  Resource,
+  envDetector,
+  processDetector,
+  hostDetector
+} = require('@opentelemetry/resources')
+
+const pkg = require('../package.json')
+
+require('dotenv').config()
+
+if (
+  !process.env.GRAFANA_OTLP_ENDPOINT ||
+  !process.env.GRAFANA_INSTANCE_ID ||
+  !process.env.GRAFANA_API_KEY
+) {
+  console.error('Grafana instrumentation not fully configured, skipping.')
```

Comment:
@ianlnf we need to change this so it works in dev when we don't instrument the app